doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -25,165 +25,51 @@ Some Key Differences
 
 .. list-table::
 
-   * - In MATLAB®, the basic data type is a multidimensional array of
-       double precision floating point numbers.  Most expressions take such
-       arrays and return such arrays.  Operations on the 2-D instances of
-       these arrays are designed to act more or less like matrix operations
-       in linear algebra.
-     - In NumPy the basic type is a multidimensional ``array``.  Operations
-       on these arrays in all dimensionalities including 2D are element-wise
-       operations.  One needs to use specific functions for linear algebra
-       (though for matrix multiplication, one can use the ``@`` operator
-       in python 3.5 and above).
-
-   * - MATLAB® uses 1 (one) based indexing. The initial element of a
+   * - In MATLAB, the basic type is a multidimensional array. By default, MATLAB
+       creates two dimensional arrays of double precision floating point numbers.
+       Operations on the 2D instances of these arrays are designed to act like
+       matrix operations in linear algebra. Most expressions take such arrays and
+       return arrays.
+     - In NumPy, the basic type is a multidimensional ``array``. By default,
+       NumPy creates 0-D or 1-D arrays for inputs of numbers or lists,
+       respectively.  Operations on these arrays in all dimensionalities
+       including 2D are element-wise operations.  One needs to use specific
+       functions for linear algebra (though for matrix multiplication, one can
+       use the ``@`` operator in Python 3.5 and above).
```

Comment:
```suggestion
       respectively.  NumPy performs operations element-by-element, so multiplying
       2D arrays with `*` is not a matrix multiplication -- it's an element-by-element 
       multiplication. (The `@` operator, available since Python 3.5, can be used
       for conventional matrix multiplication.)
       functions for linear algebra (though for matrix multiplication, one can
       use the ``@`` operator in Python 3.5 and above).
```

Reply:
Double backticks for code (operators here) in rst @bjnath